queue_services/entity-emailer/tests/unit/test_tracker.py

```diff
@@ -430,11 +430,11 @@ async def test_should_correctly_track_retries_for_failed_processing(tracker_app,
 
     # mock out process_email function to throw exception to simulate failed scenario
     with patch.object(worker, 'process_email', side_effect=QueueException('Queue Error.')):
```

Comment:
can we update this test to use `side_effect=EmailException('Queue Error.')` instead?  That will reflect what we are trying to test more accurately as it is one of the exceptions that will actually trigger retries.  

Also can we add/update a comment indicating that we are testing the max retry limit?